programs/drift/src/instructions/admin.rs

```diff
@@ -3240,6 +3332,14 @@ pub fn handle_update_perp_market_contract_tier(
     );
 
     perp_market.contract_tier = contract_tier;
+    let max_confidence_interval_multiplier =
```

Comment:
this is error prone because get_max_confidence_interval_multiplier could change it's implementation to be a diff mapping and now your cache is outta sync. 

would just store the contract tier and then look up the multiple